packages/salesforcedx-vscode-apex/package.json

```diff
@@ -174,6 +174,7 @@
     "expand-home-dir": "0.0.3",
     "find-java-home": "0.2.0",
     "path-exists": "3.0.0",
+    "shelljs": "^0.7.8",
```

Comment:
@lcampos What is the official process for bringing in new modules? Is there paperwork involved?

Reply:
Oops, just realized that it's being used elsewhere.

Reply:
@allileong no worries, but for future reference, if we were to add a completely new module we would have to see if it's already part of the list of pre-approved modules (approved by legal, etc). If it's not part of that list then we would have to go through the process of getting it approved. There's an internal link for this that I can add to our internal docs.

Reply:
@AnanyaJha why did you choose a different version than the one used on `salesforcedx-vscode-core` ?

Reply:
@lcampos I actually had the same version (`"^0.8.1"`) listed earlier, but that was what had caused the circleci build to fail so I switched it back to the current one. In the `node_modules/package.json` for `salesforcedx-vscode-apex` the version listed seemed to be `"^0.7.8"`.

 I also saw that in the package.json for `salesforcedx-vscode-core` it looks like the version in `devDependencies` is `"^0.7.8"` while the version in `dependencies` is `"^0.8.1"`.

Reply:
The problem is that you are listing shelljs twice, once in `dependencies` and again in `devDependencies`. I think the problem in circleci went away cause you now have the same version in both places but you should only have it as part of `dependencies`.

After you sort that out, let's update it to have the same versions as `salesforcedx-vscode-core` for `shelljs` and `@types/shelljs`.